client/luigi-client.d.ts

```diff
@@ -612,10 +612,19 @@ export type getEventData = () => Context;
  */
 export function getContext(): Context;
 export type getContext = () => Context;
-
+/**
+ * Sets search query parameters in Luigi Core. The search parameters will be added to the URL.
```

Comment:
Seems like there are still some instances with '**search query**' instead of '**node parameters**'.

Sets ~search query~ **node** parameters in Luigi Core. The ~search~ parameters will be added to the URL.